CryptoPkg/Library/BaseCryptLibMbedTls/Pk/CryptEc.c

```diff
@@ -418,7 +474,7 @@ EcPointInvert (
   //
   // Check input parameters.
   //
-  if ((EcGroup == NULL) || (EcPoint == NULL) || (BnCtx == NULL) ||
+  if ((EcGroup == NULL) || (EcPoint == NULL) ||

```

Comment:
please dont change BnCtx.

If there is need, please file a new issue to handle BnCtx.

Reply:
Thanks. I have restored the code. And I will submit new PR for the BnCtx.